### ISSUE TITLE
Trouble Installing

### ISSUE THREAD

**Issue by rohanpadhye** [Inventitech/strans#1]
Hi! This looks like a cool project but I'm having trouble installing it.

I tried installing via flatpak on Ubuntu and got:
```bash
$ sudo flatpak install strans.flatpak
error: The application com.inventitech.strans/x86_64/master requires the runtime org.freedesktop.Platform/x86_64/18.08 which was not found
```

I assume that being a command-line tool, strans should not require freedesktop? Or am I missing something?

I also tried building from source via `dotnet`, but got:

```bash
$ git clone https://github.com/Inventitech/strans.git
$ cd strans
$ dotnet restore
warn : The folder '/path/to/strans' does not contain a project to restore.
$ dotnet publish -c Release
'/path/to/strans' does not contain a project.json file
```
I've not really used either of these package managers before, so I'm sure I'm doing something wrong. Can someone please help?

**Comment by Inventitech** [comment]
Looks like you're missing the flatpak runtime. Have you installed it via 
`flatpak install flathub org.freedesktop.Platform//18.08 org.freedesktop.Sdk//18.08`? (Just added this to the Readme)

**Comment by rohanpadhye** [comment]
Thanks! I think I missed the step where I was supposed to add the `flathub` remote repository. Once I had that just installing `strans.flatpak` pulled in required runtimes from flathub correctly. 

Although I got strans installed system wide, I realized that I need `sudo` to be able to run it (with `sudo flatpak run com.inventitech.strans`). From reading the flatpak docs it seems that to run apps without `sudo` one must install them locally as a user. I tried doing that: `flatpak install --user strans.flatpak` which installs strans without sudo successfully. However, it still does not run without sudo.

```
$ flatpak run --user com.inventitech.strans
Error: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
$ sudo flatpak run --user com.inventitech.strans
<Works>
```

I probably need to take a flatpak 101 before attempting this again.
